recipes/Java/Date-Calendar/Calendar/Set/set-to-with-day.yaml

```
@@ -0,0 +1,29 @@
+id: scw:calendar:set-day
+version: 10
+metadata:
+  name: Migrate Calendar set() to java.time with()
+  shortDescription: Migrate Calendar set() to java.time with()
+  level: warning
+  language: java
+  enabled: true
+  descriptionFile: Java/Date-Calendar/descriptions/date-calendar.html
+  tags: java.time;java.util.Calendar;quality
+search:
+  methodcall:
+    args:
+      1:
+        referenceTo:
+          name: java.util.Calendar.DAY
```

I can't see Calendar.DAY  should this be DAY_OF_MONTH?
```suggestion
          name: java.util.Calendar.DAY_OF_MONTH
```